### PR TITLE
Feat(371) - Disabled tracks after second page

### PR DESCRIPTION
**Pull request by YuZaGa**
### PR Fixes:
- 1 Disabled tracks after second page
- 2 Removed existing implementation for hiding tracks 
- 3 Minor UI changes to RedirectToLogin component

Resolves #[371] 

### Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

File: packages/ui/src/BlogAppbar.tsx [hunk 1]
```diff
@@ -30,7 +30,7 @@ export const BlogAppbar = ({
   }, [track, problem]);
 
   let totalPages = Array.from({ length: track.problems.length }, (_, i) => i + 1);
-
+  localStorage.setItem('problemIndex', (problemIndex+1).toString());
```
Comment (hkirat) [inline packages/ui/src/BlogAppbar.tsx:33]: Why here?
Also why localStorage?

Comment (YuZaGa) [reply packages/ui/src/BlogAppbar.tsx:33]: Figured we could use problemIndex in other places so why not have it on hand in the parent component i.e BlogAppbar.

File: packages/ui/src/NotionRenderer.tsx [hunk 2]
```diff
@@ -3,34 +3,47 @@ import { NotionRenderer as NotionRendererLib } from "react-notion-x";
...
 import { useTheme } from "next-themes";
+import { useSession } from "next-auth/react";
+import RedirectToLoginCard from "./RedirectToLoginCard";
 
 // Week-4-1-647987d9b1894c54ba5c822978377910
 export const NotionRenderer = ({ recordMap }: { recordMap: any }) => {
   const { resolvedTheme } = useTheme();
+  const session = useSession();
+  const isAuthenticated = session.status === 'authenticated' && session.data !== null;
+  const storedProblemIndex = localStorage.getItem('problemIndex');
```
[the remark this answers is not in the document]
Comment (YuZaGa) [reply packages/ui/src/NotionRenderer.tsx:14]: Hmm, I missed that scenario. Mb. I can try implementing similar functionality in LessionView.tsx and pass a variable (trackVisible) to the components(Blog, CodeRenderer, MCQRenderer). That would be a better way of doing this for now and if and when new components are added

**Comment by nimit9** [comment]
Hey @YuZaGa no need to use localStorage for this. Have implemented the soln in #395

**Comment by YuZaGa** [comment]
Oh, great. LessonView implementation was exactly what I was going to go for....

**Pull request closed without merging**